Source/ACE.Entity/Position.cs

```diff
@@ -10,7 +10,17 @@
     [DbGetList("vw_character_positions", 23, "character_id")]
     public class Position
     {
-        public LandblockId LandblockId { get; set; }
+        public LandblockId LandblockId
+        {
+            get
+            {
+                return new LandblockId(Cell);

```

Comment:
I do see how creating a new object every time will be inefficient, so I will work to create private accessed variable, instead.